### PR TITLE
Order type

### PR DESCRIPTION
**Pull request by nielsvanoch**

### CODE REVIEW

File: definitions/orders/order.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ properties:
       - enabled
       - failed
       - unknown
-  type:
+  order_type:
```
Comment (freeduerinckx) [inline definitions/orders/order.yaml]: @Kjoep @daneov @joen93 can you take this change into account. This shouldn't be in use yet, if I'm not mistaken.

Comment (daneov) [reply definitions/orders/order.yaml]: It isn't used at the moment. 

I would, however, suggest to name this `type`.

Comparing `orderObject.orderType` vs `orderObject.type`, the latter feels more natural and less verbose.

Comment (joen93) [reply definitions/orders/order.yaml]: I agree with @daneov. 
Giving a model property a prefix of its model name is unnecessary, and is inconsistent with the other properties (status, id, amount, ... )

**Comment by nielsvanoch** [comment]
@joen93 @daneov I agree with your reasoning (that's why I named it `type` originally), but the problem is that that's a reserved keyword in Python. However, I see we worked around the problem in other places, so I removed that commit.

**Pull request merged**